LABWORK3/LABWORK.cpp

```diff
@@ -0,0 +1,38 @@
+﻿#include <iostream>
+#include <math.h>
+using namespace std;
+double Calc(double a, double b, double x)
+
+{   double numerator = a * pow(x, 1 / 2) - b * ((log(x) / log(4.0)));
+	double denumerator= log10(abs(x - 1));
+	return numerator/ denumerator;}
+
+
+
+double ZADANIEOne(double a, double b, double nX, double kX, double dx)
```

Comment:
ClassName - Camel case

snake_case (python) (only and recomended) 
kebab-case (not all supports)